ariadne/src/Ariadne/Config/CLI.hs

```diff
@@ -61,22 +62,22 @@ data CLI_NetworkConfigOpts = CLI_NetworkConfigOpts
     , cli_ncoPolicies        :: !(Maybe FilePath)
     , cli_ncoBindAddress     :: !(Maybe NetworkAddress)
     , cli_ncoExternalAddress :: !(Maybe NetworkAddress)
-    } deriving (Eq, Show, Generic)
+    }
 
 data CLI_CommonArgs = CLI_CommonArgs
     { cli_logConfig            :: !(Maybe FilePath)
     , cli_logPrefix            :: !(Maybe FilePath)
     , cli_reportServers        :: !(Maybe [Text])
     , cli_updateServers        :: !(Maybe [Text])
```

Comment:
You deleted `reportServers` and `updateServers`, why are they still here?

Reply:
Fair, I will check out all fields.